client/payment-methods/capability-request/capability-request-map.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { CapabilityRequestMap } from './types';
+
+const CapabilityRequestList: Array< CapabilityRequestMap > = [
+	{
+		id: 'jcb',
+		label: __( 'JCB', 'woocommerce-payments' ),
+		country: 'JP',
+		states: {
+			unrequested: {
+				status: 'info',
+				content: __(
+					'Enable JCB for your customers, the only international payment brand based in Japan.',
+					'woocommerce-payments'
+				),
+				actions: 'request',
+				actionsLabel: __( 'Enable JCB', 'woocommerce-payments' ),
+			},
+			pending_verification: {
+				status: 'warning',
+				content: __(
+					'To enable JCB for your customers, you need to provide more information.',
+					'woocommerce-payments'
+				),
+				actions: 'link',
+				actionUrl: 'http://google.com',
```

Comment:
I think the `actionUrl` is wrong.

Reply:
Adam provided the correct URL and updated it.